inputs/templates/test/ClusterBatch/ClusterBatch.json.tmpl

```diff
@@ -29,5 +29,7 @@
   "ClusterBatch.wham_vcf_tar": {{ test_batch.std_wham_vcf_tar | tojson }},
   "ClusterBatch.manta_vcf_tar": {{ test_batch.std_manta_vcf_tar | tojson }},
   "ClusterBatch.melt_vcf_tar": {{ test_batch.std_melt_vcf_tar | tojson }},
-  "ClusterBatch.ped_file": {{ test_batch.ped_file | tojson }}
+  "ClusterBatch.ped_file": {{ test_batch.ped_file | tojson }},
+
+  "ClusterBatch.outlier_cutoff_nIQR": "10000"
```

Comment:
```suggestion
  "ClusterBatch.outlier_cutoff_nIQR": "8"
```
10000 is used in FilterBatch to effectively disable sample filtering. Here we should make it something statistically reasonable, otherwise the cutoffs at +/- 10000 IQR make the plots unreadable.

Reply:
I've been using 6 for plotting